news/2023/2023-07-05-beatmap-pack-new-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
```

Comment:
the file name needs to match the title, so `2023-07-{day}-introducing-new-beatmap-pack-categories.md`